integration-tests/bot/chainbot.py

```diff
@@ -340,7 +341,16 @@ def coin_to_voting_power(coin):
         return addrs
 
 
-async def gen_genesis(cfg):
+async def gen_genesis(cfg_orig):
+    cfg = copy.deepcopy(cfg_orig)
+    newnodes= []
+    for n in cfg["nodes"]:
+        print(n)
+        if n["bonded_coin"]> 0:
+            newnodes.append(n)
+    cfg["nodes"]= newnodes
```

Comment:
It's simpler to filter the nodes out with `bonded_coin == 0` in `app_state_cfg` when constructing `council_nodes`.

Reply:
it's intended operation, i wan to add node which is not validator, but has big amount of bonds.

Reply:
Isn't this part of code try to remove the node with `bonded_coin == 0`?